File: get_feature8.js
const sqlite3 = require('sqlite3');
const db = new sqlite3.Database('./features.db');

db.get("SELECT id, category, name, description, steps, passes, in_progress, dependencies, priority FROM features WHERE id = 8", (err, row) => {
  if (err) {
    console.error(err);
  } else if (row) {
    console.log(JSON.stringify(row, null, 2));
  } else {
    console.log(JSON.stringify({error: "Feature #8 not found"}));
  }
  db.close();
});

File: bin/get_feature8.dart
// ignore_for_file: avoid_print

import 'package:sqflite_common_ffi/sqflite_ffi.dart';

void main() async {
  sqfliteFfiInit();
  databaseFactory = databaseFactoryFfi;

  final db = await databaseFactory.openDatabase('features.db');

  final List<Map> results = await db.query(
    'features',
    where: 'id = ?',
    whereArgs: [8],
  );

  if (results.isNotEmpty) {
    final feature = results.first;
    print('ID: ${feature['id']}');
    print('Category: ${feature['category']}');
    print('Name: ${feature['name']}');
    print('Description: ${feature['description']}');
    print('Steps: ${feature['steps']}');
    print('Passing: ${feature['passes']}');
    print('In Progress: ${feature['in_progress']}');
    print('Priority: ${feature['priority']}');
  } else {
    print('Feature #8 not found');
  }

  await db.close();
}
